build/sources.props

```diff
@@ -5,6 +5,7 @@
     <RestoreSources>$(DotNetRestoreSources)</RestoreSources>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       $(RestoreSources);
+      https://www.myget.org/F/30de4ee06dd54956a82013fa17a3accb;
```

Comment:
Regular MyGet feeds aren't super reliable. Also, we should be using the v3 protocol if we do reference a feed.

Reply:
@natemcmaster is dotnet.myget.org/gallery/aspnetcore-tools a viable feed to mirror this package to?

Reply:
Depends. Where is the source code for this package?

Reply:
@Eilon says it's probably in TFS, @dougbu might know more.

Reply:
Can we copy this code into aspnet/BuildTools? I'd like to avoid adding a dependency to a thing that we don't know how to recreate. Also, build-from-source....

Reply:
What is the package of interest here?

Reply:
microsoft.web.skipstrongnames

Reply:
Oddly the package says that comes from the old ASP.NET Web Stack site on CodePlex. But, I can't see anything in TFS for it. I'll look around a few other places.

Reply:
All this does is set \ remove a registry key under a specific location. If it's important to have sources for this rather than consume it as a binary, it'll be easier to just rewrite it as a task.

Reply:
Gallery shows v3 source is https://www.myget.org/F/30de4ee06dd54956a82013fa17a3accb/api/v3/index.json

Reply:
Yeah I talked to @ryanbrandenburg yesterday. If we can't find the source, let's just decompile w/ ILSpy because I figure the whole app is like a 50-line `Program.Main()` and nothing more (it just reads an XML file and does a `for()` loop to 'register' each assembly/public key token). Whether we do it as a task or a tool is up to whoever wants to write it.